src/main/java/com/redhat/devtools/lsp4ij/internal/telemetry/TelemetryManager.java

```diff
@@ -76,22 +78,24 @@ private LanguageServerDefinitionListener createListener() {
             @Override
             public void handleAdded(@NotNull LanguageServerAddedEvent event) {
                 if (project.equals(event.getProject())) {
```

Comment:
project matters here

Reply:
Since telemetry manager is a singleton, I have removed this condition and project field.